src/Symfony/Bridge/Twig/Resources/views/Email/zurb_2/notification/body.html.twig

```diff
@@ -0,0 +1,63 @@
+{% filter inky_to_html|inline_css %}
+<html>
+<head>
+<style>
+    {% block style %}
+        {{ source("@email/zurb_2/main.css") }}
+        {{ source("@email/zurb_2/notification/local.css") }}
+    {% endblock %}
+</style>
+</head>
+<body>
+<spacer size="32"></spacer>
+<container class="body_{{ ("urgent" == importance ? "alert" : ("high" == importance ? "warning" : "default")) }}">
+    <spacer size="16"></spacer>
+    <row>
+        <columns large="12" small="12">
+            {% block lead %}
+                <small><strong>{{ importance|upper }}</strong></small>
+                <p class="lead">
+                    {{ email.subject }}
+                </p>
+            {% endblock %}
+
+            {% block content %}
+                {% if markdown %}
+                    {{ include('@email/zurb_2/notification/content_markdown.html.twig') }}
```

Comment:
shouldn't this template be named `@email/default/notification/content_markdown.html.twig` instead?

Reply:
No, we are really in the `zurb_2` theme here.